projects/SDE Lighthouse 04 - Bruce/template-sql/2.medications.template.sql

```diff
@@ -24,14 +24,14 @@ SELECT
     CAST("MedicationDate" AS DATE) AS "MedicationDate",
     "SuppliedCode",
     "Dosage",
-    "Quantity",
+ --   "Quantity",
     "Units"
 FROM INTERMEDIATE.GP_RECORD."GP_Medications_SecondaryUses" gp
 INNER JOIN {{cohort-table}} c ON gp."FK_Patient_ID" = c."FK_Patient_ID"
 WHERE "SuppliedCode" IN (SELECT code FROM {{code-set-table}} WHERE concept in ('belimumab','hydroxychloroquine','chloroquine'));
 
```

Comment:
could you include the end date in the meds file and only include meds up to the end date?